app/models/orchestration.rb

```diff
@@ -6,14 +6,14 @@ class Orchestration < ApplicationRecord
   class OrchestrationAlreadyRan < StandardError; end
   class OrchestrationOngoing < StandardError; end
 
-  enum kind: [:bootstrap, :upgrade, :removal, :force_removal]
+  enum kind: [:bootstrap, :upgrade, :migration, :removal, :force_removal]
```

Comment:
Adding `migration` here will confuse rails, because it automatically assigned `bootstrap = 0`, `upgrade = 1`, `removal = 2`, `force_removal = 3`. Either we do add it at the end (`[:bootstrap, :upgrade, :removal, :force_removal, :migration]`), or we convert this array into a hash and do something like:

```ruby
enum kind: { bootstrap: 0, upgrade: 1, migration: 4, removal: 2, force_removal: 3 }
```

Otherwise if we keep this as is, old `removal` orchestrations saved in the database (as `2`) will be identified as `migration`, and old `force_removal` orchestrations saved with (as `3`) will be identified as `removal`.